flybywiresim/a32nx
# Creating "Bandit Mask" livery version

## Summary of Changes
This PR adds another livery implementing the A350 "bandit mask" over the FBW A320 Neo livery. Just few cents to this project I fell in love with!

## Screenshots (if necessary)


Thumbnails:
![thumbnail](https://user-images.githubusercontent.com/5606511/103824474-450f6f80-5074-11eb-9453-9b35f57cb78c.jpg)
![thumbnail](https://user-images.githubusercontent.com/5606511/103824463-3f198e80-5074-11eb-8332-40f77ac477e6.jpg)

Screenshots:
![20210106235506_1](https://user-images.githubusercontent.com/5606511/103829844-4c864700-507b-11eb-97d0-de5653df0542.jpg)
![20210106235326_1](https://user-images.githubusercontent.com/5606511/103829886-4f813780-507b-11eb-8b4d-173ac1eea183.jpg)
![20210106235312_1](https://user-images.githubusercontent.com/5606511/103830157-6162da80-507b-11eb-8ad8-0cf93d82aab5.jpg)
![20210106234323_1](https://user-images.githubusercontent.com/5606511/103830338-6e7fc980-507b-11eb-93f7-c9d46b6bfc99.jpg)


## References
- A350 and A330 "Bandit mask" (officially Shades) livery: https://www.airbus.com/newsroom/press-releases/en/2015/11/shades-livery-now-also-on-new-a330neo.html
- A320 Neo SAS livery with "Bandit mask": https://community.infiniteflight.com/t/scandinavian-airlines-sas-airbus-a320neo-new-livery/383583

## Additional context
Few cents of mine to a totally awesome project!
Discord username (if different from GitHub): Slidspitfire

## Testing instructions
- Download the A32NX artifact
- Install it as usual
- Should be P&P with Flight Simulator


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
